charter/index.html

```diff
@@ -381,13 +387,11 @@ <h2>Success Criteria</h2>
 
     <!-- Testing and interop -->
 	  <p>In order to advance to
-		  <a href="https://www.w3.org/Consortium/Process/#RecsPR" title="Proposed Recommendation">Proposed Recommendation</a>, each normative specification is expected to have
-		  <a href="https://www.w3.org/Consortium/Process/#implementation-experience">at least two independent interoperable
+		  <a href="https://www.w3.org/policies/process/#RecsPR" title="Proposed Recommendation">Proposed Recommendation</a>, each normative specification is expected to have
+		  <a href="https://www.w3.org/policies/process/#implementation-experience">at least two independent interoperable
 			  implementations</a> of every feature defined in the specification, where
 interoperability can be verified by passing open test suites, and two or
-more implementations interoperating with each other. In order to advance to
-Proposed Recommendation, each normative specification must have an open
-test suite of every feature defined in the specification.</p>
+more implementations interoperating with each other.</p>
 	  <p>There should be testing plans for each specification, starting from the earliest drafts.</p>
     <p>To promote interoperability, the group will document testing policies, processes, and procedures, taking <a href="https://solidproject.org/ED/qa">Solid QA</a> as an input.</p>
```

Comment:
This line already mentions Solid QA as an input; is it intended to have it stated twice?

Reply:
As mentioned in the original comment and in the commit:

>Explicitly list Solid QA as input under Other Deliverables as per Success Criteria.

They are stated in different context. Since Other Deliverables already covers Test Suite and Implementation Report, it is useful, and doesn't hurt, to list as as input document.